features/validation_requests/managing_validation_requests.feature

```diff
@@ -0,0 +1,21 @@
+Feature: Managing validation requests
+  Background:
+    Given I am logged in as an assessor
```

Comment:
So I ran into this dilemma: we usually use personas and have files like `andre_the_assessor_does_blah` but because we have two different roles (assessor, reviewer) I was wondering if we should really use that approach this time? My worry being having to duplicate a tons of features to make sure we cover both roles (which we don't do anyway).

I looked on the internet and I couldn't find a solid ground rule for naming feature files. I stole the idea of personas from the e-petitions repo, so I thought we could try naming feature files after the actual feature? 

What do you think?